apps/webapp/src/components/Connection/ConnectionTable.tsx

```
@@ -71,68 +72,100 @@
+                  className="h-7 gap-1"
+                  type="submit"
+                  onClick={() => window.open(link, "_blank")}
+                >
+                  <p className="mr-2">Open Link</p>
+                  <svg
+                    width="15"
+                    height="15"
+                    viewBox="0 0 15 15"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)